apps/judicial-system/backend/migrations/20211220090749-update-case.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const replaceEnum = require('sequelize-replace-enum-postgres').default
+
+module.exports = {
+  up: (queryInterface) => {
+    // replaceEnum does not support transactions
+    return replaceEnum({
+      queryInterface,
+      tableName: 'case',
+      columnName: 'session_arrangements',
+      newValues: [
+        'ALL_PRESENT',
+        'ALL_PRESENT_SPOKESPERSON',
+        'PROSECUTOR_PRESENT',
+      ],
```

Comment:
This migration will fail because some cases have "REMOTE_SESSION" assigned to "session_arrangements".

I suggest starting the migration by replacing "REMOTE_SESSION" entries with "PROSECUTOR_PRESENT". That way, the enum modification will work and the rest of the code changes will make sense, including pdf document generation.

Reply:
Awesome! Done :)